modules/ROOT/pages/metadata-propagation-between-flows.adoc

```diff
@@ -0,0 +1,60 @@
+= Metadata Propagation Between Flows
```

Comment:
After reading through and editing the existing text, I finally came to understand these points:

- In Anypoint Studio 7.10, you can add a Flow Reference component to one flow that automatically propagates the metadata in that flow (_actual_ metadata) to all other flows it references, as _custom_ (or _expected_) metadata, so that you don't have to add the actual metadata to the referenced flows manually.
- This enables you to scaffold your flows using an inheritance model that automatically adds or updates custom metadata throughout your app.
- However, any subsequent change to actual metadata automatically overwrites the previously propagated custom/expected metadata; if you do not want this to happen, you must remove the component (?) and manually customize the metadata in referenced flows.

Is this correct?
The current use of synonymous names for types of flows and metadata confused me, so I think some term definition in the context of the intro narrative might help.
A graphic might help, too.

Reply:
I've left in the original edits, just so you can see how I experienced the text in a realtime reading flow.
Please let me know if chatting about this would be easier.  :-)